.github/workflows/check-codestyle-all.yml

```diff
@@ -2,11 +2,12 @@ name: Check Code Style
 on:
   push:
     branches: [main, staging]
+    pull_request:
```

Comment:
is this intentional? did you want `on.pull_request` instead of `on.push.pull_request`?

Reply:
Ah yeah good catch